leaflet-geoman.d.ts

```diff
@@ -722,6 +722,24 @@ declare module 'leaflet' {
             /** If true, vertex removal that cause a layer to fall below their minimum required vertices will remove the entire layer. If false, these vertices can't be removed. Minimum vertices are 2 for Lines and 3 for Polygons (default:true). */
             removeLayerBelowMinVertexCount?: boolean;
 
+            /** Defines which layers should dragged with this layer together. true syncs all layers in the same LayerGroup(s) or you pass an `Array` of layers to sync. (default:false). */
+            syncLayersOnDrag?: [] | boolean;
```

Comment:
@ryan-morris is this correct? It can be a boolean or an array of Layers,LayerGroups, FeatureGroups,GeoJSON Group,...

Reply:
If you know  all the allowed types to be in the array you can declare it like: 

`syncLayersOnDrag?: (L.Layer|L.FeatureGroup|L.GeoJSON)[] | boolean;`

if it's very open for what it will accept then you want:

`syncLayersOnDrag?: any[] | boolean;`

just specifying `[]` results in 

```
const a: L.PM.EditModeOptions = {
    syncLayersOnDrag: [layer,layer,featureGroup],
};

// error: Type 'Layer' is not assignable to type 'undefined'.ts(2322)
```

Reply:
Ok I understand. 
L.FeatureGroup, L.LayerGroup are extending from L.Layer, so does it works if I change it to `L.Layer[] | boolean`?

Reply:
Yes